Wikimate.php

```diff
@@ -255,6 +353,50 @@ public function login($username, $password, $domain = null)
 		return true;
 	}
 
+	/**
+	 * Gets the current value for the maxlag parameter.
```

Comment:
This feels more natural to me, but feel free to ignore if you think otherwise.

```suggestion
	 * Gets the current value of the maxlag parameter.
```